jetty-core/jetty-ee/jetty-ee-test-resources/src/main/java/org/eclipse/jetty/ee/test/resources/TestEeResources.java

```diff
@@ -0,0 +1,86 @@
+//
+// ========================================================================
+// Copyright (c) 1995 Mort Bay Consulting Pty Ltd and others.
+//
+// This program and the accompanying materials are made available under the
+// terms of the Eclipse Public License v. 2.0 which is available at
+// https://www.eclipse.org/legal/epl-2.0, or the Apache License, Version 2.0
+// which is available at https://www.apache.org/licenses/LICENSE-2.0.
+//
+// SPDX-License-Identifier: EPL-2.0 OR Apache-2.0
+// ========================================================================
+//
+
+package org.eclipse.jetty.ee.test.resources;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.net.URL;
+import java.nio.file.FileSystem;
+import java.nio.file.FileSystemNotFoundException;
+import java.nio.file.FileSystems;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Map;
+
+public class TestEeResources
+{
+    public static URL getResource(String name)
+    {
+        return TestEeResources.class.getResource(name);
+    }
+
+    public static InputStream getResourceAsStream(String name)
+    {
+        return TestEeResources.class.getResourceAsStream(name);
+    }
+
+    public static Path getResourceAsPath(String name)
+    {
+        URL url = TestEeResources.class.getResource(name);
+        if (url == null)
+            return null;
+        try
+        {
+            URI uri = url.toURI();
+            // Handle resources from JAR
+            if ("jar".equals(uri.getScheme()))
+            {
+                try
+                {
+                    FileSystem fileSystem;
+                    try
+                    {
+                        fileSystem = FileSystems.getFileSystem(uri);
+                    }
+                    catch (FileSystemNotFoundException e)
+                    {
+                        fileSystem = FileSystems.newFileSystem(uri, Map.of());
```

Comment:
This `fileSystem` isn't closed anywhere.
Hey, the build is green, so ... uh ... no problem?

Perhaps this should just use the Resource layer and a test specific ResourceFactory.Closable that closes the opened resources at test end?
The fact that we return Path is just a secondary behavior IMO.

Reply:
My thinking was that this approach might be used to test things like the ResourceHandler, so I didn't want to use Resources to test Resources.
Also, when I did at first use a Resource, it of course was detected in the leak detection, as it wasn't closed.    So I think manually pulling this and never closing is OK for a test.  I'll comment as much.'